command_line_tool.py

```diff
@@ -0,0 +1,37 @@
+import argparse
+import corn_app.contour
+import corn_app.mask
+import os
+import cv2
+import json
+import sys
+
+json_data = ""
+photo_dir = ""
+output_dir = ""
+
+def main():
+	for file in os.listdir(photo_dir):
+		if file.endswith('.JPG'):
+			image = cv2.imread(os.path.join(photo_dir, file))
+			masked_image = corn_app.mask.mask_yellow(image)
```

Comment:
Could there be a command line option to save the masked_image result also? I assume it's something Ikhide would find useful.